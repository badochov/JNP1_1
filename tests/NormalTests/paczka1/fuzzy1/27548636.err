Error in line 7: OeDvqyP S769 25,9
Error in line 16: A921 A921 63,82
Error in line 24: A921 A921 262,79
Error in line 25: xiW A584 405,7
Error in line 27: xiW A910 36,0
Error in line 28: xiW S769 93,7
Error in line 14: 2TrcOq0JI A910 26,6
Error in line 33: FJLWsZ8vJFC S769 34,1
