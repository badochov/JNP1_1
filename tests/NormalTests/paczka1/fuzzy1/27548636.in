8x9Gd A910 46,7
FJLWsZ8vJFC A910 93,5
8x9Gd A910 127,5
FJLWsZ8vJFC A910 413,3
xiW A921 7,1
8x9Gd A910 24,5
OeDvqyP S769 25,9
OeDvqyP A921 12,4
OeDvqyP A921 35,6
xiW A921 28,5
8x9Gd A910 16,1
8x9Gd A910 40,6
xiW A910 48,4
2TrcOq0JI A910 26,6
FJLWsZ8vJFC A584 84,9
A921 A921 63,82
OeDvqyP A910 7,3
?
FJLWsZ8vJFC A584 39,9
?8x9Gd
xiW A910 216,8
OeDvqyP A910 10,4
OeDvqyP A921 79,9
A921 A921 262,79
xiW A584 405,7
OeDvqyP A921 11,2
xiW A910 36,0
xiW S769 93,7
xiW A910 97,0
2TrcOq0JI A921 70,2
A921 A584 39,8
xiW A910 60,1
FJLWsZ8vJFC S769 34,1
FJLWsZ8vJFC A921 91,2
FJLWsZ8vJFC A921 426,8
?xiW
OeDvqyP A921 31,2
2TrcOq0JI A921 82,1
2TrcOq0JI A910 46,5
?S769