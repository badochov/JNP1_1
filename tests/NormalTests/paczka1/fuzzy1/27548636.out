8x9Gd A 89,2
FJLWsZ8vJFC A 319,8
OeDvqyP A 23,2
xiW A 21,4
A910 409,0
A921 44,6
8x9Gd A 89,2
xiW A 226,7
